Rules/Warns/Zombie_WarnsCommon.as

```diff
@@ -0,0 +1,206 @@
+// SonantDread @ October 11th 2024
+#define SERVER_ONLY
```

Comment:
Common files should be SERVER_ONLY or CLIENT_ONLY as they will carry over into any file they are put into and cause potential issues.